docs/source/zh/trainer.md

```diff
@@ -0,0 +1,411 @@
+<!--Copyright 2023 The HuggingFace Team. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+the License. You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+
+⚠️ Note that this file is in Markdown but contain specific syntax for our doc-builder (similar to MDX) that may not be
+rendered properly in your Markdown viewer.
+
+-->
+
+# Trainer
+在Transformers 库中，[`Trainer`] 提供了一个用于 PyTorch 模型的训练和评估完整训练器。您只需传递必要的训练部件（模型、tokenizer、数据集、评价函数、训练超参数等），[`Trainer`] 类会处理剩下的事情。这使得模型训练变得更加简易，而无需每次手动编写自己的训练循环。与此同时，[`Trainer`] 是非常灵活的，并提供了大量的训练选项，因此您可以根据自己的训练需求进行自定义。
+
+<Tip>
+
+除了 [`Trainer`] 类之外，Transformers 还提供了 [`Seq2SeqTrainer`] 类，用于序列到序列的任务，比如翻译或摘要。还有来自[TRL](https://hf.co/docs/trl) 库的 [`~trl.SFTTrainer`] 类，它封装了 [`Trainer`] 类，并进行了优化，用于训练 Llama-2 和 Mistral 等自回归技术的语言模型。[`~trl.SFTTrainer`] 还支持其他特性，如序列打包、LoRA、量化和 DeepSpeed，可有效扩展到任何模型大小。
+
+<br>
+
+随时查看其他 [`Trainer`] 类型的[API reference](./main_classes/trainer)参考，以了解何时使用哪种类型。总体而言，[`Trainer`] 是最通用的选择，并适用于广泛的任务。[`Seq2SeqTrainer`] 适用于序列到序列的任务，[`~trl.SFTTrainer`] 适用于训练语言模型。
+
+</Tip>
+
+在开始之前，请确保已安装 [Accelerate](https://hf.co/docs/accelerate) - 一个用于在分布式环境中启用和运行 PyTorch 训练的库。
+
+```bash
+pip install accelerate
+
+# upgrade
+pip install accelerate --upgrade
+```
+
+本指南提供了 [`Trainer`] 类的概述。
+
+## 基本用法
+[`Trainer`] 包括您在基本训练循环中的所有代码：
+
+1. 执行训练步骤以计算损失
+2. 使用 [`~accelerate.Accelerator.backward`] 方法计算梯度
+3. 基于梯度更新权重
+4. 重复此过程，直到达到预定的轮数
+
+[`Trainer`] 类将所有这些代码抽象出来，因此您无需每次手动编写训练循环，或者如果您刚开始使用 PyTorch 进行训练并不知道该怎么做。您只需提供用于训练所需的基本组件（如模型和数据集），[`Trainer`] 类会处理其他一切。
+
+如果您想指定任何训练选项或超参数，可以在 [`TrainingArguments`] 类中找到它们。例如，让我们定义将模型保存在`output_dir`中，并在训练后将模型推送到Hub，使用`push_to_hub=True`。
+
+```py
+from transformers import TrainingArguments
+
+training_args = TrainingArguments(
+    output_dir="your-model",
+    learning_rate=2e-5,
+    per_device_train_batch_size=16,
+    per_device_eval_batch_size=16,
+    num_train_epochs=2,
+    weight_decay=0.01,
+    evaluation_strategy="epoch",
+    save_strategy="epoch",
+    load_best_model_at_end=True,
+    push_to_hub=True,
+)
+```
+
+将 `training_args`与模型、数据集、用于预处理数据集的内容（根据您的数据类型，可能是tokenizer、feature extractor或image processor）、数据收集器，以及用于计算训练期间想要跟踪的指标的函数一起传递给 [`Trainer`]。
+
+最后，调用 [`~Trainer.train`] 开始训练！
+
+```py
+from transformers import Trainer
+
+trainer = Trainer(
+    model=model,
+    args=training_args,
+    train_dataset=dataset["train"],
+    eval_dataset=dataset["test"],
+    tokenizer=tokenizer,
+    data_collator=data_collator,
+    compute_metrics=compute_metrics,
+)
+
+trainer.train()
+```
+
+### checkpoints
```

Comment:
Should this be translated?

```suggestion
### checkpoints
```

Reply:
In my opinion, translating checkpoints into Chinese is not as easy to understand as checkpoints.

Reply:
Just like the tokenizer